templates/patterns/dropdown/dropdown.ui_patterns.yml

```diff
@@ -1,71 +1,66 @@
 dropdown:
-  label: "Dropdown"
-  description: "Toggle contextual overlays for displaying lists of links and more with the Bootstrap dropdown plugin. https://v5.getbootstrap.com/docs/5.0/components/dropdowns/"
+  label: 'Dropdown'
+  description: 'Toggle contextual overlays for displaying lists of links and more with the Bootstrap dropdown plugin. For more information visit: https://v5.getbootstrap.com/docs/5.0/components/dropdowns/'
   settings:
     link:
       type: boolean
-      label: link
-      description: Show as a link
-      preview: false
+      label: 'link'
+      description: 'When enabled, shows dropdown as a link. False by default.'
     direction:
       type: select
-      label: Direction
+      label: 'Direction'
+      description: 'Sets the direction of the dropdown:'
       options:
-        dropdown: "dropdown"
-        dropup: "dropup"
-        dropend: "dropend"
-        dropstart: "dropstart"
+        dropdown: 'dropdown (default)'
+        dropup: 'dropup'
+        dropend: 'dropend'
+        dropstart: 'dropstart'
     alignment:
       type: select
-      label: Alignment
+      label: 'Alignment'
+      description: 'Sets the alignment of the dropdown menu (if not set, then will be automatically positioned 100% from the top and along the left side of its parent):'
       options:
-        dropdown-menu-end: "dropdown-menu-end"
-        dropdown-menu-sm-end: "dropdown-menu-sm-end"
-        dropdown-menu-md-end: "dropdown-menu-md-end"
-        dropdown-menu-lg-end: "dropdown-menu-lg-end"
-        dropdown-menu-xl-end: "dropdown-menu-xl-end"
-        dropdown-menu-xxl-end: "dropdown-menu-xxl-end"
-        dropdown-menu-start: "dropdown-menu-start"
-        dropdown-menu-sm-start: "dropdown-menu-sm-start"
-        dropdown-menu-md-start: "dropdown-menu-md-start"
-        dropdown-menu-lg-start: "dropdown-menu-lg-start"
-        dropdown-menu-xl-start: "dropdown-menu-xl-start"
-        dropdown-menu-xxl-start: "dropdown-menu-xxl-start"
+        dropdown-menu-end: 'dropdown-menu-end'
+        dropdown-menu-sm-end: 'dropdown-menu-sm-end'
+        dropdown-menu-md-end: 'dropdown-menu-md-end'
+        dropdown-menu-lg-end: 'dropdown-menu-lg-end'
+        dropdown-menu-xl-end: 'dropdown-menu-xl-end'
+        dropdown-menu-xxl-end: 'dropdown-menu-xxl-end'
+        dropdown-menu-start: 'dropdown-menu-start'
+        dropdown-menu-sm-start: 'dropdown-menu-sm-start'
+        dropdown-menu-md-start: 'dropdown-menu-md-start'
+        dropdown-menu-lg-start: 'dropdown-menu-lg-start'
```

Comment:
Too complex, we don't need all of this. IF, and it's a big if, is ever needed, these actually map to classes and they can be passed together with attributes.